plugins/inputs/bigbluebutton/bigbluebutton.go

```diff
@@ -0,0 +1,167 @@
+package bigbluebutton
+
+import (
+	"encoding/xml"
+	"fmt"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type BigBlueButton struct {
+	URL         string `toml:"url"`
+	APIEndpoint string `toml:"api_endpoint"`
+	SecretKey   string `toml:"secret_key"`
+	ServerName  string `toml:"server_name"`
+}
+
+var bbbConfig = `
+	## Required BigBlueButton server url
+	url = "http://localhost:8090"
+
+	## Required BigBlueButton api endpoint
+	api_endpoint = "/bigbluebutton/api/"
```

Comment:
I guess this is the default path in a default install? If so, it seems a sensible default and you should comment the line and use the value as a default.

Reply:
Hi. Is there a process for default variable or I just set it in the code?

Reply:
There are two options. You can either check for an empty variable `APIEndpoint` in `Init()` and then initialize with the default _or_ you set the default in `init()` when constructing the `BigBlueButton` struct like:
```golang
func init() {
	inputs.Add("bigbluebutton", func() telegraf.Input {
		return &BigBlueButton{
			APIEndpoint: "/bigbluebutton/api/",
		}
	})
￼}
```
The former has the benefit of also covering the case where the user sets an empty endpoint...